readme.txt

```diff
@@ -0,0 +1,44 @@
+=== The Events Calendar Extension: Calendar Customizer ===
+Contributors: ModernTribe
+Donate link: http://m.tri.be/29
+Tags: events, calendar, customize
+Requires at least: 4.5
+Tested up to: 4.9.6
+Requires PHP: 5.6
+Stable tag: 1.0.0
+License: GPL version 3 or any later version
+License URI: https://www.gnu.org/licenses/gpl-3.0.html
+
+Customize the looks of The Events Calendar's month view.
+
+== Description ==
+
+This extension adds more than a dozen formatting options to the WordPress customizer that will enable you to customize the looks of the month view in The Events Calendar. You will find the settings under Themes > Customize > The Events Calendar > Month View.
+
+== Installation ==
+
+Install and activate like any other plugin!
+
+* You can upload the plugin zip file via the *Plugins ‣ Add New* screen
+* You can unzip the plugin and then upload to your plugin directory (typically _wp-content/plugins)_ via FTP
+* Once it has been installed or uploaded, simply visit the main plugin list and activate it
+
+== Frequently Asked Questions ==
+
+= Where can I find more extensions? =
+
+Please visit our [extension library](https://theeventscalendar.com/extensions/) to learn about our complete range of extensions for The Events Calendar and its associated plugins.
+
+= What if I experience problems? =
+
+We're always interested in your feedback and our [premium forums](https://theeventscalendar.com/support-forums/) are the best place to flag any issues. Do note, however, that the degree of support we provide for extensions like this one tends to be very limited.
+
+== Known Limitations ==
+
+* Clearing a setting that has a set value might not always show the correct change in the live preview.
+
+== Changelog ==
+
+= 1.0.0 2018-06-25 =

```

Comment:
Borrowing from our core plugins, the format we use is `= [1.2.3] 2020-12-31 =` (ie, wrap the version number in square brackets).